rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/DataUtility.java

```diff
@@ -104,14 +104,18 @@ public static FilteredTopWorkflow toFilteredTopWorkflowWithIssues(FilteredTopWor
                                        filteredTopWorkflowData.getNumberOfExecutions());
     }
 
-    public static WorkflowExecutionTime
-            toFilteredTopWorkflowWithExecutionTime(WorkflowExecutionTimeData filteredTopWorkflowData) {
+    public static WorkflowExecutionTime toWorkflowExecutionTime(WorkflowExecutionTimeData filteredTopWorkflowData) {
         return new WorkflowExecutionTime(filteredTopWorkflowData.getProjectName(),
                                          filteredTopWorkflowData.getWorkflowName(),
                                          filteredTopWorkflowData.getExecutionTime(),
                                          filteredTopWorkflowData.getNumberOfExecutions());
     }
 
+    public static CompletedJobsCount toCompletedJobsCount(CompletedJobsCountData completedJobsCountData) {
+        return new CompletedJobsCount(completedJobsCountData.getJobsWithIssues(),
+                                      completedJobsCountData.getJobsWithIssues());
```

Comment:
Is it normal you add the same param twice ?

Reply:
no, I will fix it. Thanks!